### ISSUE TITLE
Find bug tracking tools instead of bugzilla 

### ISSUE THREAD

**Issue by ZeinaDaghlas** [sallyishak/SoftwareConstructionProject#1]
we don't have virtual machine to install bugzilla 
Need a new tool .

**Comment by ZeinaDaghlas** [comment]
Hi Remal , 

Could you please describe your problem with bugzilla ??


Thanks 
Zeina

**Comment by remalsaad** [comment]
Yes, H have a problem in downloading it  since I didn't find an online version.

**Comment by ZeinaDaghlas** [comment]
Hi Team , 

we agreed to discuss this issue tom
